win_configure/projects.VS2008/scheduler.vcproj

```diff
@@ -307,6 +307,54 @@
 					/>
 				</FileConfiguration>
 			</File>
+			<File
```

Comment:
Do the header files need to be included? I don't know if this is for building or for displaying in the IDE.

Reply:
If I am doing something wrong in the project files, please tell me what to do.  I'm somewhat clueless about these files.